http/src/main/resources/reference.conf

```diff
@@ -205,9 +205,9 @@ azure {
         type = "CustomScript",
         version = "2.1",
         minor-version-auto-upgrade = true,
-        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/61f9ad246382ade9a5242e4c6f899633876767f7/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
+        file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/83bbaddc45f042e5b151b0dfcac98a6fd3559f37/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
```

Comment:
Can we put this image hash into a variable with a app-wide scope so that we can use the same value here and n the specs?

Reply:
Something like that would be nice. I'd say out of scope for this ticket, I'm just copying previously established conventions.